suite-native/module-device-settings/src/screens/BackupAndPassphrase/BackupAndPassphraseScreen.tsx

```diff
@@ -0,0 +1,39 @@
+import { useSelector } from 'react-redux';
+
+import {
+    selectIsDeviceBackupUnfinished,
+    selectIsDeviceInitialized,
+    selectIsPortfolioTrackerDevice,
+} from '@suite-common/wallet-core';
+import { VStack } from '@suite-native/atoms';
+import { useDeviceConnectionGuard } from '@suite-native/device-authorization';
+import { Translation } from '@suite-native/intl';
+import { DynamicScreenHeader, Screen } from '@suite-native/navigation';
+
+import { CheckBackupCard } from './CheckBackupCard';
+import { PassphraseCard } from './PassphraseCard';
+
+export const BackupAndPassphraseScreen = () => {
+    const isPortfolioTrackerDevice = useSelector(selectIsPortfolioTrackerDevice);
+    const isDeviceInitialized = useSelector(selectIsDeviceInitialized);
+    const isDeviceBackupUnfinished = useSelector(selectIsDeviceBackupUnfinished);
+    const isCheckBackupAvailable = isDeviceInitialized && !isDeviceBackupUnfinished;
+
+    useDeviceConnectionGuard();
```

Comment:
nit: Wouldn't it be more explicit inside BackupAndPassphraseStackNavigator? It will work the same though so feel free to resolve.